content/en/docs/configuration/acme/_index.md

```diff
@@ -110,21 +117,92 @@ spec:
             key: apikey
       selector:
         matchLabels:
-          use-cloudflare-solver: "true"
+        - "use-cloudflare-solver": "true"
+        - "email": "user@example.com"
+```
+
+#### DNS Names
+
+The `dnsNames` selector is used to define the exact DNS challenges so solve.
+This means that `Certificates` containing any of these DNS names will be
+selected.  If a match is found, a `dnsNames` selector will take precedence over
+a [`dnsZones`](./#dns-zones) selector. If multiple solvers match with the same
+`dnsNames` value, the solver with the most matching labels in
+[`matchLabels`](./#match-labels) will be selected. If neither has more matches,
```

Comment:
Can we omit the `./` prefix here and just link direct to `#match-labels` as this is in the same doc?